<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Просмотр заказа</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="4">
    <widget class="QTableWidget" name="orders_table"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>381</width>
       <height>61</height>
      </size>
     </property>
     <property name="title">
      <string>Поиск</string>
     </property>
     <widget class="QLineEdit" name="search_e">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="search_cb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>231</width>
       <height>111</height>
      </size>
     </property>
     <property name="title">
      <string>Фильтрация</string>
     </property>
     <widget class="QDateEdit" name="date1">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="date2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="date">
       <date>
        <year>2099</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>От</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>До</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox1">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>20</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Принят</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>40</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выполнен</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkbox3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>70</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменён</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearFil_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>211</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить фильтр</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="3" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>111</height>
      </size>
     </property>
     <property name="title">
      <string>Сортировка</string>
     </property>
     <widget class="QPushButton" name="sortsum_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сумма заказа ↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sortdate_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Дата выдачи ↑</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sort_clear_b">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Сброс</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="info_b">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Просмотр заказа</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="excel_b">
     <property name="text">
      <string>Отчёт Excel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QPushButton" name="back_menu_b">
     <property name="text">
      <string>В меню</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
